Formula/questdb.rb

```diff
@@ -2,7 +2,7 @@ class Questdb < Formula
   desc "Time Series Database"
   homepage "https://questdb.io"
   url "https://github.com/questdb/questdb/releases/download/6.2.1/questdb-6.2.1-no-jre-bin.tar.gz"
-  sha256 "607d95346841846e015e6723af25b4047dc1acdf1912d0a259ffff15c314b93e"
+  sha256 "4c7411c5585c4a2d39994a28004dd0f2fa3fba2c8c42d01a8ac3777e1bcca02e"
```

Comment:
Please contact upstream to ask if they retagged, because otherwise their GitHub might have been compromised

Reply:
I'm in touch with folks who cut the release, and will report back!

Reply:
I did QuestDB release on 28/02/2022 and had to re-build the tag and re-publish binaries on 01/03/2022 because a bash file inside the release ended up with Windows line endings.

So re-built binaries uploaded to the github. It seems that the brew release captured the first build of 6.2.1 in the hash signature, it would be good to update the signature here.

Reply:
@ideoma: thanks for the insight!
@SMillerDev: please let me know if this is enough of a justification.

Reply:
Yeah, that's fine. We generally assume malice when tags change because the [git manual](https://git-scm.com/docs/git-tag#_on_re_tagging) says re-tagging is "the insane thing" to do. So better to be safe than sorry.